src/foam/u2/stack/Stack.js

```diff
@@ -82,6 +90,33 @@ foam.CLASS({
       this.stack_.length = this.depth;
       this.stack_[pos] = [v, parent, opt_id];
       this.pos = pos;
+    },
+    function setToNullCurrentMemento() {
```

Comment:
Please add documentation, which you can do with a /** comment **/ in the first line of the function.

Reply:
fixed